functions/security/idtoken.go

```diff
@@ -27,14 +26,21 @@ import (
 	"google.golang.org/api/idtoken"
 )
 
-// makeGetRequest makes a request to the provided targetURL with an authenticated client.
-func makeGetRequest(w io.Writer, targetURL string) error {
-	// functionURL := "https://TARGET_URL"
+// `makeGetRequest` makes a request to the provided `targetURL`
+// with an authenticated client using audience `audience`.
+//
+// [END functions_bearer_token]
+// Example `audience` value (Cloud Run): https://my-cloud-run-service.run.app/
+// [END cloudrun_service_to_service_auth]
+// [START functions_bearer_token]
+// Example `audience` value (Cloud Functions): https://project-region-projectid.cloudfunctions.net/myFunction
+// [START cloudrun_service_to_service_auth]
+func makeGetRequest(w io.Writer, targetURL string, audience string) error {
```

Comment:
This context may be in the accompanying docs, but in all uses I see here, targetURL and audience are identical. I'd suggest including an explanation of when these args might differ, for other readers without context.

Reply:
Fixed.